.github/workflows/reusable-release.yml

```diff
@@ -48,6 +48,7 @@ jobs:
       - name: Checkout out repo
         uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3
         with:
+          ref: ${{ inputs.release_tag }}
```

Comment:
I believe this is fine if it's not specified, it's just be empty and default to pulling from HEAD.